docs/pages/api.rst

```diff
@@ -59,30 +59,32 @@ and *period type* options.
 
 .. code-block:: python
 
-    import wetterdienst
+    import wetterdienst import DWDObservationSites
```

Comment:
from wetterdienst import DWDObservationSites

Reply:
Fixed. Thanks for spotting this.